game_eggs/steamcmd_servers/arma/arma3/egg-arma3.json

```diff
@@ -1,60 +1,60 @@
 {
-    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "_comment": "Pterodactyl Arma 3 Egg ~ David Wolfe (Red-Thirten) ~ 2021-07-11",
     "meta": {
         "version": "PTDL_v1",
-        "update_url": null
+        "update_url": "https:\/\/github.com\/parkervcp\/eggs\/blob\/master\/game_eggs\/steamcmd_servers\/arma\/arma3\/egg-arma3.json"
```

Comment:
update_url feature does not exist at this moment. It's a placeholder for the future when and if such feature will be implemented

Reply:
Understood, but does it hurt to be there? I figured it would be good to include just incase the feature is ever implemented. Please reply if you would formally like it removed.

Reply:
We don't know how it will work, as such this could break it or show a bad example for others looking at the egg